routes/my_api.js

```diff
@@ -54,6 +58,22 @@ router.use((req, res, next) => {
     next();
 });
 
+router.post('/sms', (req, res, next) => {
+    Promise.resolve().then(async () => {
+        let phone = `${req.body.From}`
+        let message = `${req.body.Body}`;
```

Comment:
In our code style, if we want to convert something to string, we do `String(req.body.From)`. But I assume those properties are already strings?